code/modules/projectiles/guns/projectile/constructable/flamethrower.dm

```diff
@@ -205,6 +205,8 @@
 		onclose(user, "flamethrower", src)
 
 /obj/item/weapon/gun/projectile/flamethrower/Topic(href,href_list[])
+	if (..())
+		return
```

Comment:
Yes, it works fine, but it fucks up if you have a child that has additional behavior in `Topic()`
If you put this exact check in a child, it wouldn't work unless this returns 1